auxdir/x_ac_json.m4

```diff
@@ -33,11 +33,11 @@ AC_DEFUN([X_AC_JSON], [
         for d in $_x_ac_json_dirs; do
           test -d "$d" || continue
           test -d "$d/include" || continue
-          test -f "$d/include/json-c/json_object.h" || test -f "$d/include/json/json_object.h" || continue
+          test -f "$d/include/json-c12/json_object.h" || test -f "$d/include/json/json_object.h" || continue
```

Comment:
does this work on el8? i don't think so

Reply:
Hmm, should be the other way around. Also, I do _not_ recall changing anything ever in this file, but it did build.

Reply:
this is correct for el7 (which requires josn-c12); but not on el8, where it is called json-c

Reply:
@itkovian ouch. this one still needs to be addressed at some point (but only relevant for burstbuffer and restd)